<launch>

  <!-- <include file="$(find speech_processing)/launch/followme.launch"/> -->

  <node name="main_state" pkg="main_state" type="main.py" output="screen" args="TestFollowLeg" />
  <!-- <node ns="people" name="people_detection" pkg="people_detection" type="people_detection_node"> -->
  </node>
</launch>
